tests/assets/widerface_dataset/dataset_meta.json

```diff
@@ -1,3 +1,3 @@
 {
-"label_map": {"0": "Parade", "1": "Handshaking"}
```

Comment:
Why these 2 labels were removed?

Reply:
The labels in the label_map represent the category of the images like of Parade, Handshaking, Dancing etc right? If we were to include this line, the "labels" should be all these above including the face label. As these labels array indicate the labels of the annotations bbox which is only face, I've removed this line. Also, as the import process actually works without this file. Should I include them and push the new changes?

Reply:
Hmm, it seems like it should be updated for the whole format, not just this example. I agree that there should only be 1 label, which name can be specified in the labels file, with the default value of `face`. 

To fully support what was supposed in the current implementation, we could introduce an import/export parameter like `label_as_prefix`. The scene prefix, such as `0--Parade` might be included in the dataset item id, and should be prepended as a default value on export if none is specified. During import, we check this parameter, and use either the scene prefix or the `face` label (possibly mapped in the `labels.txt` / `dataset_meta`). During export, we check this parameter, and either prepend the file name and dir with the bbox label, or leave it as is and just write the label name in the `labels.txt` / `dataset_meta`. 

Probably, it would be quite a big change for a bug fix, so I suggest to stick with what's done in the current PR at the moment.